Server/src/server/dataCenter/DataCenter.java

```diff
@@ -333,6 +339,20 @@ public void readAllCards() {
         GameServer.serverPrint("Original Cards Loaded");
     }
 
+    public void readAllCustomCards(){
+        String customCardsPath = Config.getInstance().getPathToCustomCards();
+        File[] files = new File(customCardsPath).listFiles();
+            if (files != null) {
```

Comment:
is this null check necessary?   Intuitively, I would image listFiles would return an emptylist, not null.   You would need to doublecheck the documentation.

Reply:
"If this abstract pathname does not denote a directory, then this method returns null. Otherwise an array of File objects is returned, one for each file or directory in the directory. "

So it's necessary.